paasta_tools/api/views/instance.py

```diff
@@ -138,3 +139,20 @@ def instance_status(request):
         raise InstanceFailure(error_message, 500)
 
     return instance_status
+
+
+@view_config(route_name='service.instance.tasks', request_method='GET', renderer='json')
+def instance_tasks(request):
+    status = instance_status(request)
+    task_id = request.swagger_data.get('task_id', None)
+    slave_hostname = request.swagger_data.get('slave_hostname', None)
+    try:
+        mstatus = status['marathon']
+    except KeyError:
+        raise InstanceFailure("Only marathon tasks supported", 400)
+    tasks = get_tasks_from_app_id(mstatus['app_id'], slave_hostname=slave_hostname, task_id=task_id)
+    tasks = [{'task_id': task['id'],
```

Comment:
I think I actually tried to do this but pyramid (and or swagger) doesn't let us have optional params in the path. So it'd actually have to be something like: `/service/{service}/instance/{instance}/tasks` and a second endpoint: `/service/{service}/instance/{instance}/tasks/task/{task_id}`

Which seems worse? As for the filtering, just seemed easier to return what we need for now but I could be convinced either way.